src/test_workflow/compare_benchmark/compare_test_runner.py

```diff
@@ -0,0 +1,62 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+import os
+import subprocess
+
+from test_workflow.compare_benchmark.compare_args import CompareArgs
+
+
+class CompareTestRunner:
+    def __init__(self) -> None:
+        self.container_name = "compare-container"
+
+    def run_comparison(self, compare_args: CompareArgs) -> None:
+        """
+        Perform the comparison between two test results using the OSB tool.
```

Comment:
Use `OpenSearch-Benchmark` instead of OSB here.

Reply:
nit: `opensearch-benchmark`